pkg/scheduler/internal/cache/cache.go

```diff
@@ -736,19 +751,6 @@ func (cache *schedulerCache) expirePod(key string, ps *podState) error {
 	return nil
 }
 
-// GetNodeInfo returns cached data for the node name.
```

Comment:
I think 2 functions in `pkg/scheduler/internal/cache/fake/fake_cache.go` can also be removed:

- func (c *Cache) GetNodeInfo(nodeName string) (*v1.Node, error)
- func (c *Cache) ListNodes() []*v1.Node

Reply:
Done